### DISCUSSION TITLE
Access to the raw value

### DISCUSSION THREAD

**Discussion by Pommfriz** [jomjol/AI-on-the-edge-device#1957] [Ideas]
Instead of installing MQTT, NodeRed, Grafana or something else, it would be nice to provide a website on a specific port
which only contains the measured value, nothing else.
So it would be very easy to read the value using curl and process it own applications without any overhead.

**Comment by jurasjo** [comment]
Have you seen this?:
https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/#value

`http://<IP>/value`

Below is the whole content of the page:
```
<html>
<script type="text/javascript" id="custom-useragent-string-page-script">
</script>
<head>
</head>
<body>68.491
</body>
</html>
```

**Reply by caco3** [reply]
easiest is to use the `/json` entrypoint
